### ISSUE TITLE
Where is os.environ["SSL_COM_USERNAME"] supposed to be set?

### ISSUE THREAD

**Issue by mfcallahan** [system76/thelio-io-windows#10]
I was able to clone the repo and successfully follow the instructions up until the `build.py` script. I am receiving the following error:

```
> python build.py
...
info: installing component 'rustfmt'
error: no such subcommand: `wix`

        Did you mean `fix`?
Traceback (most recent call last):
  File "build.py", line 16, in <module>
    subprocess.check_call([
  File "C:\Python310\lib\subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cargo', 'wix', '--nocapture', '--verbose']' returned non-zero exit status 101.
PS C:\Users\matth\GitHub\thelio-io-windows>
```

I found this similar issue, not sure if it is related: [system76/thelio-io-windows/issues/8](https://github.com/system76/thelio-io-windows/issues/8), however I also do not see the file `target/wix/thelio-io-0.1.0-x86_64.msi` anywhere, so I'm not sure how to proceed. I've tried with Python 3.9 and 3.10, both installed globally and for individual user.

Is this still the latest and greatest for installing Thelio-IO drivers under Windows?

**Comment by mfcallahan** [comment]
I was able to resolve the original issue in my comment above, based on what I read [here](https://volks73.github.io/cargo-wix/cargo_wix/index.html) about installing the Wix toolset. The `build.py` script had a few bugs for me, but I was able to work through those and run the script up until [line 47](https://github.com/system76/thelio-io-windows/blob/264b41a1fc7e02a76cd7da93fa34728506b503f5/build.py#L47) which has the comment "Sign with specified cloud signing key." But being unfamiliar with this tool, I'm not sure where these values are supposed to be set, they all throw ad "KeyError" exception as they don't exist:

```py
os.environ["SSL_COM_CREDENTIAL_ID"],
os.environ["SSL_COM_USERNAME"],
os.environ["SSL_COM_PASSWORD"],
os.environ["SSL_COM_TOTP_SECRET"],
```

I did not see anything around configuring the code signing tool but it appears that I am missing these critical things. How can I resolve this?

**Comment by jackpot51** [comment]
You probably do not need code signing if you are installing it yourself.

**Comment by mfcallahan** [comment]
What is the typical workflow for someone wanting to install Thelio IO drivers in Windows on their own machine, without the need for code signing?  I'm experimenting with Windows on my new (Intel) Thelio Mira. I saw in the other issue I linked to that it looks like you're still very much in-progress with Windows 11, so I'll stay tuned! I'm happy to help test anything, provide feedback/notes from my installation experience, etc.
